src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -52,10 +55,9 @@ class BlogController extends Controller
      * @Route("/", name="admin_post_index")
      * @Method("GET")
      */
-    public function indexAction()
+    public function indexAction(EntityManagerInterface $em, UserInterface $user = null)
     {
-        $entityManager = $this->getDoctrine()->getManager();
-        $posts = $entityManager->getRepository(Post::class)->findBy(['author' => $this->getUser()], ['publishedAt' => 'DESC']);
+        $posts = $em->getRepository(Post::class)->findBy(['author' => $user], ['publishedAt' => 'DESC']);
```

Comment:
Also I prefer `$this->getDoctrine()->getRepository()` shortcut if `$em` is not needed :)